src/main/java/org/dependencytrack/event/kafka/processor/ProcessorInitializer.java

```diff
@@ -40,6 +40,9 @@ public void contextInitialized(final ServletContextEvent event) {
         PROCESSOR_MANAGER.registerProcessor(RepositoryMetaResultProcessor.PROCESSOR_NAME,
                 KafkaTopics.REPO_META_ANALYSIS_RESULT, new RepositoryMetaResultProcessor());
 
+        PROCESSOR_MANAGER.registerProcessor(EpssMirrorProcessor.PROCESSOR_NAME,
+                KafkaTopics.NEW_EPSS, new EpssMirrorProcessor());
```

Comment:
Properties will need to be added to `application.properties` to configure the new processor:

https://github.com/DependencyTrack/hyades-apiserver/blob/91bc41ecddaf0bec40edd3e3e987a3ed089d111e/src/main/resources/application.properties#L458-L517